lib/ddtrace/contrib/active_record/patcher.rb

```diff
@@ -93,7 +99,31 @@ def self.sql(_name, start, finish, _id, payload)
           span.start_time = start
           span.finish(finish)
         rescue StandardError => e
-          Datadog::Tracer.log.error(e.message)
+          Datadog::Tracer.log.debug(e.message)
+        end
+
+        def self.instantiation(_name, start, finish, _id, payload)
+          span = get_option(:tracer).trace(
+            'active_record.instantiation',
+            resource: payload.fetch(:class_name),
+            span_type: Datadog::Ext::Ruby::TYPE
+          )
+
+          # Inherit service name from parent, if available.
```

Comment:
👍 I think this is a better default. In general we expect this to be child of something else, though the corner case is well covered with `active_record` default. Furthermore, if someone wants to change and split the service, they can. So we provide a reasonable default that is easy to configure.